### PR TITLE
Bump travis php 7 4 27

### PR DESCRIPTION
**Pull request by righdforsa**
Bumping PHP version for Travis tests

### Fixed Issues
$ Related to https://github.com/Expensify/Expensify/issues/179679

### Web Tests
What areas need to be tested for regressions?
n/a

### Mobile Tests
If this affects mobile, what needs to be tested to validate your changes? 
n/a

### CODE REVIEW

File: .travis.yml-e [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: php
```
Comment (luacmartins) [inline .travis.yml-e:1]: Does this file need to be committed? Looking at previous version bump PRs I don't see us updating it.

**Review by luacmartins — COMMENTED** [review]
Left a comment

File: .travis.yml-e (hunk 1 above)
Comment (righdforsa) [reply .travis.yml-e:1]: That's weird, no, I should delete, I think.

**Review by luacmartins — APPROVED** [review]

**Comment by righdforsa** [comment]
K, bumped the version, checks are running now.

**Comment by righdforsa** [comment]
Done, ready for re-approval @luacmartins

**Review by luacmartins — APPROVED** [review]

**Pull request merged**